samples/tflm_ethosu/README.md

```diff
@@ -0,0 +1,33 @@
+# Arm(R) Ethos(TM)-U Tensorflow Lite for Microcontrollers test application
```

Comment:
This needs to be converted to .rst so it shows up in https://docs.zephyrproject.org/latest/samples/index.html.

See https://docs.zephyrproject.org/latest/contribute/documentation/index.html for details on contributing documentation.

Reply:
I have never used rst before, but I hope I managed to correctly translate markdown into ReStructuredText.

Reply:
> I have never used rst before, but I hope I managed to correctly translate markdown into ReStructuredText.

You can check for yourself. I have just approved the run of a full CI on your PR. When the "Documentation build" step in the below list of CI checks is complete, you can click on the "details" link for that check, and the "summary" link on the resulting page. That should give you a link to a full build of the HTML documentation for your PR. Then you can verify that it is as you expect. If you want to build the documentation locally, you can do so using the instructions in the link I gave above, although this does take some time.

Please let me know when you are satisfied that your documentation renders correctly as HTML or if you have any specific questions about problems you had with your .rst.